show/vnet.py

```diff
@@ -237,10 +238,13 @@ def all():
     for k in vnet_rt_keys:
         r = []
         r.extend(k.split(":", 2)[1:])
+        state_db_key = '|'.join(k.split(":",2))
         val = appl_db.get_all(appl_db.APPL_DB, k)
+        val_state = state_db.get_all(state_db.STATE_DB, state_db_key)
         r.append(val.get('endpoint'))
         r.append(val.get('mac_address'))
         r.append(val.get('vni'))
+        r.append(val_state.get('state'))
```

Comment:
Can you check the output if the entry is not present in state_db?

Reply:
done